cernopendata/modules/fixtures/data/records/jade-data.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "abstract": {
+      "description": "<p>JADE real data</p>",
```

Comment:
Please expand the text greatly. Since we shall have only one record for the JADE tape data, we need a very detailed description here.